File: examples/customer-address-jpa/src/main/java/io/zenwave360/example/core/outbound/jpa/CustomerRepository.java
package io.zenwave360.example.core.outbound.jpa;

import io.zenwave360.example.core.domain.Customer;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

/** Spring Data JPA repository for the Customer entity. */
@SuppressWarnings("unused")
@Repository
public interface CustomerRepository extends JpaRepository<Customer, Long> {}

File: examples/kustomer-address-jpa/src/main/kotlin/io/zenwave360/examples/kotlin/repository/jpa/CustomerRepository.kt
package io.zenwave360.examples.kotlin.repository.jpa

import io.zenwave360.examples.kotlin.domain.*
import java.math.*
import java.time.*
import java.util.*

import org.springframework.data.jpa.repository.*
import org.springframework.stereotype.Repository

/**
 * Spring Data JPA repository for the Customer entity.
 */
@Suppress("unused")
@Repository
interface CustomerRepository : JpaRepository<Customer, Long> {
}
